handler/step.go

```diff
@@ -65,10 +65,12 @@ func HandleStartStep(config *config.Config) http.HandlerFunc {
 			if v.Name == "harness" {
 				v.Name = hv.HostPath.Name
 				v.Path = hv.HostPath.Path
+				v.ContainerPath = hv.HostPath.ContainerPath
```

Comment:
You might also need to set `s.StartStepRequestConfig.WorkingDir` if `ContainerPath` is not empty

Reply:
That actually messes up Run step, since it's running on the host and relies on WorkingDir.

Reply:
`ContainerPath` should only be populated to when step is running in the container